package.json

```diff
@@ -2,7 +2,7 @@
   "name": "detect-adblock",
   "version": "0.0.1",
   "description": "Implements detecting of adblockers in browser",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
fwiw, node modules should not have to be manually compiled upon installation, they should either have a dist folder with a built version of the module ready to go or there should be a `postinstall` hook in the `package.json` that triggers the build command upon installation, like:

```
"scripts": {
    "postinstall": "npm run build"
} 
```